package.json

```diff
@@ -10,12 +10,10 @@
     "build:browser": "browserify -d -s minim -t [ babelify ] -o dist/minim.js dist/lib/minim.js",
     "coverage": "istanbul cover _mocha -- -R spec --recursive --require babel-register",
     "coveralls": "coveralls <coverage/lcov.info",
-    "frontendtests": "karma start",
     "lint": "eslint .",
-    "nodejstests": "mocha -R spec --recursive",
     "precoveralls": "npm run coverage",
-    "prepublish": "npm run lint",
-    "test": "npm run nodejstests && npm run frontendtests"
+    "test": "mocha -R spec --recursive",
```

Comment:
I'm not fan of short options in written form as it's bad documentation, but it's not a sin I'd block the PR for

Reply:
Me neither, updated in 776a7db (where I just removed the -R option as spec is default) and move --recursive to mocha.opts